src/main/java/core/basesyntax/service/InvalidUserException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class InvalidUserException extends RuntimeException {
+    InvalidUserException(String massage) {
```

Comment:
The constructor for `InvalidUserException` should be public to allow instantiation from other packages, especially since this is a custom exception that might be used outside of the `service` package.